nfs-ganesha/build/build_deb

```diff
@@ -29,18 +30,19 @@ mkdir build
 cd build
 
 # make source tarball
-cmake -DCMAKE_BUILD_TYPE=Maintainer $WORKSPACE/nfs-ganesha/src && make dist || exit 0
+cmake -DCMAKE_BUILD_TYPE=Maintainer $WORKSPACE/nfs-ganesha/src && make dist || exit 1
+
+# terrible hack
```

Comment:
why is this a terrible hack? mind adding more context here so someone can maybe pick this up and improve?